### ISSUE TITLE
Campaign

### ISSUE THREAD

**Issue by shariquerik** [frappe/crm#111]

**Comment by maiphuongfw** [comment]
@shariquerik  Is this feature used for email campaigns? After creating email lists by using filters and sorting, then users can create and send an email campaign directly to this list in Frappe CRM instead of using another email marketing service. Is that what you are picturing? I think this flow step is exceptionally easy to use for standard users

**Comment by joshuarestivo** [comment]
I would like to see this feature, too. It would work exactly as the stock CRM EMail Campaign functionality does, although I'd hope that it's more intuitive to configure and manage. A user would create an 'EMail Campaign' that allows scheduling of a series of email messages to an Email Group. The addresses in an Email Group are **not** considered Leads as salespeople often obtain lists of hundreds of addresses. The vast majority of them never respond or the addresses simply don't work. They don't want to pollute their Leads list with a lot of noise. If someone responds to one of the Campaign messages, then it should be possible to easily convert their email address to a Lead.

**Comment by federicocalvo** [comment]
I think this is one of those functions that is worth investing time in, not least because Frappe Inc itself uses it.

The idea of keeping something simple, like the old CRM already has, is an incredible thing for SMEs. While it could be thought of a little better (it's not really designed to streamline the current bell functions in ERPNext), it's not necessary to have a mailchimp.

Being able to generate campaigns, simple templates as ERPNext currently has today (plain text, html, markdown), but above all being able to clarify them or group them easily. As an extra data would be the info that can be collected from these campaigns in the future (clicks, opens, etc).

**Comment by federicocalvo** [comment]
https://github.com/The-Commit-Company/mailrun

**Comment by shariquerik** [comment]
![image_2024-07-08_16-25-04](https://github.com/frappe/crm/assets/30859809/08247b89-be53-4762-8810-c0cbb019f3d4)

Point 1 & 2 already exist in ERPNext CRM

**Comment by federicocalvo** [comment]
> Point 1 & 2 already exist in ERPNext CRM

We are aware of that, but I think those of us who post comments on this issue understand that this CRM replaces the ERPNext CRM.
Although it could easily be another app (campaign).

**Comment by shariquerik** [comment]
> Although it could easily be another app (campaign).

I agree but it will slow me down, let me built it for Frappe CRM first later we can separate it out.

**Comment by shariquerik** [comment]
> Although it could easily be another app (campaign).

I agree but it will slow me down, let me built it for Frappe CRM first later we can separate it out.

**Comment by 1ubuntuuser** [comment]
We have tried using the current email campaigns, but it is a bit limited. For example, it only sends emails at midnight. I was about to start some code to upgrade it but it might be efforts are better placed in this one if it is going to replace it.

**Comment by 1ubuntuuser** [comment]
Is anyone working on this currently? Don't want to duplicate any work.

**Comment by shariquerik** [comment]
> Is anyone working on this currently? Don't want to duplicate any work.

Currently no one is working on it

**Comment by 1ubuntuuser** [comment]
@shariquerik Thanks, I'll start playing with it. Is the plan to port erpnext CRM code to CRM? So for example will doctype "Campaign Email Schedule" be moved to CRM from ERPNext?

**Comment by michellealva** [comment]
@pratikb64 we can try an email campaign - along with follow up actions, and if possible SM campaigns like LinkedIn and Twitter too.
ref: https://docs.google.com/document/d/1qGkTTqqR_2ktI3HCxO5bwYUK9_WcD_Hu4kbEEs4WsHM/edit?tab=t.0

**Comment by the3dfox** [comment]
Wuhuu so this should be the first official app built with Studio. Can't wait

**Comment by michellealva** [comment]
@the3dfox , yes seems like there'll be some delay. Will post an update once we have some clarity.
